refactor: update text wrapping behavior for PPC messages

jadutter opened the pull request:
Change PPC text messages to be on the same line as the logo, and wrap to a newline when there is not enough space

jadutter committed:
feat: update text wrapping behavior for PPC messages

Change PPC text messages to be on the same line as the logo, and wrap to a newline when there is not enough space

jadutter committed:
refactor: restrict text wrap to alternative logo

jadutter committed:
refactor: customize breakpoints

yeul commented on server/locale/US/PPC/mutations/ni_non-us.js:
```diff
@@ -112,7 +113,8 @@ export default {
                     styles: [
                         messageDisclaimerMediaQuery(breakpointCalc - 1),
```
Let's remove this line: ``` messageDisclaimerMediaQuery(breakpointCalc - 1),``` since it's no longer applicable with our new wrapping logic.

yeul commented on server/locale/US/PPC/mutations/niq.js:
```diff
@@ -151,7 +153,8 @@ export default {
                     altContentMediaQuery(textSize * 33),
-                    `.message__logo-container { width: ${textSize * 5}px }`
+                    `.message__logo-container { width: ${textSize * 5}px }`,
+                    textWrap(textSize * 30, textSize, 'US')
```
Let's up the multiplier to be ```textSize * 40``` so that the wrapping starts a little sooner.

jadutter replied:
for this line, or for this file?

yeul commented on server/locale/US/PPC/mutations/niq.js:
```diff
@@ -151,7 +153,8 @@ export default {
                 styles: [
                     basicMediaQuery(textSize * 18.5),
                     altContentMediaQuery(textSize * 33),
```
Let's increase this to be `altContentMediaQuery(textSize * 42)`

jadutter committed:
Remove textWrap from 'logo.type:alternative && logo.position:top' for all files
Add style 'logo.type:alternative && logo.position:top' for ezp_any_gtz and ezp_any_eqz
Minor css adjustment to 'alternative' and 'alternative && right' for niq_non-us, ni_non-us, pala_multi_gtz
Minor css adjustments to niq and ni_non-us

jadutter committed:
Change sizing for niq_non-us
Minor css change to PMG 'alternative && right'

Seavenly commented:
:tada: This PR is included in version 1.15.3 :tada:

The release is available on [GitHub release](https://github.com/paypal/paypal-messaging-components/releases/tag/v1.15.3)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
